yt_dlp/extractor/unsupported.py

```diff
@@ -164,11 +164,15 @@ class KnownPiracyIE(UnsupportedInfoExtractor):
         r'viewsb\.com',
         r'filemoon\.sx',
         r'hentai\.animestigma\.com',
+        r'thisav\.com',
     )
 
     _TESTS = [{
         'url': 'http://dood.to/e/5s1wmbdacezb',
         'only_matching': True,
+    }, {
+        'url': 'https://thisav.com/en/terms',
```

Comment:
I suggested this url, but why don't we actually just use /favicon.ico for all the piracy sites (eg https://thisav.com/favicon.ico)? That url exists for every website, and stores just the site's logo (which I highly doubt would be copyright, and it's not really our problem if it is)